src/service/email/EmailService.ts

```diff
@@ -0,0 +1,33 @@
+const nodemailer = require('nodemailer');
+const jwt = require('jsonwebtoken')
+import { inviteEmailContent } from '@messages/email/content'
+
+export const EmailService = () => {
+  const send = (from, subject, email, content) => {
```

Comment:
Se estamos usando tipo, talvez essa seria uma oportunidade bacana de explorar o sistema de tipos do typescript:
```suggestion
type EmailPayload = {
  from: string;
  subject: string;
  email: string;
  content: string;
}

export const EmailService = () => {
  const send = ({from, subject, email, content}: EmailPayload) => {
```

Se não estamos usando tipos, ignorem este comentário completamente.